### ISSUE TITLE
When a play a track the total time is same to current time (Progess Bar)

### ISSUE THREAD

**Issue by SpeakerBlack** [googledrive/drive-music-player#105]

**Comment by Gribnif** [comment]
I agree that this is a bug. It seems to happen mostly with .mp3 files.

A workaround is to use the Space bar to temporarily stop playback. Then, when you press it again, the progress bar usually fixes itself.
